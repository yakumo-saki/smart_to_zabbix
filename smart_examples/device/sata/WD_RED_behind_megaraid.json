{
    "json_format_version": [
      1,
      0
    ],
    "smartctl": {
      "version": [
        7,
        2
      ],
      "svn_revision": "5155",
      "platform_info": "x86_64-linux-5.11.22-5-pve",
      "build_info": "(local build)",
      "argv": [
        "smartctl",
        "-a",
        "--json",
        "/dev/bus/0",
        "-d",
        "megaraid,0"
      ],
      "messages": [
        {
          "string": "Warning: This result is based on an Attribute check.",
          "severity": "warning"
        }
      ],
      "exit_status": 4
    },
    "device": {
      "name": "/dev/bus/0",
      "info_name": "/dev/bus/0 [megaraid_disk_00] [SAT]",
      "type": "sat+megaraid,0",
      "protocol": "ATA"
    },
    "model_family": "Western Digital Red",
    "model_name": "WDC WD30EFRX-68EUZN0",
    "serial_number": "***************",
    "wwn": {
      "naa": 5,
      "oui": 5358,
      "id": 28697753090
    },
    "firmware_version": "80.00A80",
    "user_capacity": {
      "blocks": 5860533168,
      "bytes": 3000592982016
    },
    "logical_block_size": 512,
    "physical_block_size": 4096,
    "rotation_rate": 5400,
    "trim": {
      "supported": false
    },
    "in_smartctl_database": true,
    "ata_version": {
      "string": "ACS-2 (minor revision not indicated)",
      "major_value": 1022,
      "minor_value": 0
    },
    "sata_version": {
      "string": "SATA 3.0",
      "value": 62
    },
    "interface_speed": {
      "max": {
        "sata_value": 14,
        "string": "6.0 Gb/s",
        "units_per_second": 60,
        "bits_per_unit": 100000000
      },
      "current": {
        "sata_value": 3,
        "string": "6.0 Gb/s",
        "units_per_second": 60,
        "bits_per_unit": 100000000
      }
    },
    "local_time": {
      "time_t": 1636099108,
      "asctime": "Fri Nov  5 16:58:28 2021 JST"
    },
    "smart_status": {
      "passed": true
    },
    "ata_smart_data": {
      "offline_data_collection": {
        "status": {
          "value": 130,
          "string": "was completed without error",
          "passed": true
        },
        "completion_seconds": 40020
      },
      "self_test": {
        "status": {
          "value": 0,
          "string": "completed without error",
          "passed": true
        },
        "polling_minutes": {
          "short": 2,
          "extended": 401,
          "conveyance": 5
        }
      },
      "capabilities": {
        "values": [
          123,
          3
        ],
        "exec_offline_immediate_supported": true,
        "offline_is_aborted_upon_new_cmd": false,
        "offline_surface_scan_supported": true,
        "self_tests_supported": true,
        "conveyance_self_test_supported": true,
        "selective_self_test_supported": true,
        "attribute_autosave_enabled": true,
        "error_logging_supported": true,
        "gp_logging_supported": true
      }
    },
    "ata_sct_capabilities": {
      "value": 28733,
      "error_recovery_control_supported": true,
      "feature_control_supported": true,
      "data_table_supported": true
    },
    "ata_smart_attributes": {
      "revision": 16,
      "table": [
        {
          "id": 1,
          "name": "Raw_Read_Error_Rate",
          "value": 200,
          "worst": 200,
          "thresh": 51,
          "when_failed": "",
          "flags": {
            "value": 47,
            "string": "POSR-K ",
            "prefailure": true,
            "updated_online": true,
            "performance": true,
            "error_rate": true,
            "event_count": false,
            "auto_keep": true
          },
          "raw": {
            "value": 1350,
            "string": "1350"
          }
        },
        {
          "id": 3,
          "name": "Spin_Up_Time",
          "value": 210,
          "worst": 176,
          "thresh": 21,
          "when_failed": "",
          "flags": {
            "value": 39,
            "string": "POS--K ",
            "prefailure": true,
            "updated_online": true,
            "performance": true,
            "error_rate": false,
            "event_count": false,
            "auto_keep": true
          },
          "raw": {
            "value": 4466,
            "string": "4466"
          }
        },
        {
          "id": 4,
          "name": "Start_Stop_Count",
          "value": 100,
          "worst": 100,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 756,
            "string": "756"
          }
        },
        {
          "id": 5,
          "name": "Reallocated_Sector_Ct",
          "value": 200,
          "worst": 200,
          "thresh": 140,
          "when_failed": "",
          "flags": {
            "value": 51,
            "string": "PO--CK ",
            "prefailure": true,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 7,
          "name": "Seek_Error_Rate",
          "value": 200,
          "worst": 200,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 46,
            "string": "-OSR-K ",
            "prefailure": false,
            "updated_online": true,
            "performance": true,
            "error_rate": true,
            "event_count": false,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 9,
          "name": "Power_On_Hours",
          "value": 13,
          "worst": 13,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 63823,
            "string": "63823"
          }
        },
        {
          "id": 10,
          "name": "Spin_Retry_Count",
          "value": 100,
          "worst": 100,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 11,
          "name": "Calibration_Retry_Count",
          "value": 100,
          "worst": 100,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 12,
          "name": "Power_Cycle_Count",
          "value": 100,
          "worst": 100,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 750,
            "string": "750"
          }
        },
        {
          "id": 192,
          "name": "Power-Off_Retract_Count",
          "value": 200,
          "worst": 200,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 669,
            "string": "669"
          }
        },
        {
          "id": 193,
          "name": "Load_Cycle_Count",
          "value": 200,
          "worst": 200,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 86,
            "string": "86"
          }
        },
        {
          "id": 194,
          "name": "Temperature_Celsius",
          "value": 111,
          "worst": 102,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 34,
            "string": "-O---K ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": false,
            "auto_keep": true
          },
          "raw": {
            "value": 39,
            "string": "39"
          }
        },
        {
          "id": 196,
          "name": "Reallocated_Event_Count",
          "value": 200,
          "worst": 200,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 197,
          "name": "Current_Pending_Sector",
          "value": 200,
          "worst": 200,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 198,
          "name": "Offline_Uncorrectable",
          "value": 200,
          "worst": 200,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 48,
            "string": "----CK ",
            "prefailure": false,
            "updated_online": false,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 199,
          "name": "UDMA_CRC_Error_Count",
          "value": 200,
          "worst": 199,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 3,
            "string": "3"
          }
        },
        {
          "id": 200,
          "name": "Multi_Zone_Error_Rate",
          "value": 200,
          "worst": 200,
          "thresh": 0,
          "when_failed": "",
          "flags": {
            "value": 8,
            "string": "---R-- ",
            "prefailure": false,
            "updated_online": false,
            "performance": false,
            "error_rate": true,
            "event_count": false,
            "auto_keep": false
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        }
      ]
    },
    "power_on_time": {
      "hours": 63823
    },
    "power_cycle_count": 750,
    "temperature": {
      "current": 39
    },
    "ata_smart_error_log": {
      "summary": {
        "revision": 1,
        "count": 0
      }
    },
    "ata_smart_self_test_log": {
      "standard": {
        "revision": 1,
        "table": [
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 28352
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 28185
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 28017
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 27849
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 27681
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 27513
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 27345
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 23993
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 23825
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 23658
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 23490
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 23322
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 23154
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 22986
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 22819
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 22651
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 22483
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 22315
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 22147
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 21979
          },
          {
            "type": {
              "value": 1,
              "string": "Short offline"
            },
            "status": {
              "value": 0,
              "string": "Completed without error",
              "passed": true
            },
            "lifetime_hours": 21812
          }
        ],
        "count": 21,
        "error_count_total": 0,
        "error_count_outdated": 0
      }
    },
    "ata_smart_selective_self_test_log": {
      "revision": 1,
      "table": [
        {
          "lba_min": 0,
          "lba_max": 0,
          "status": {
            "value": 0,
            "string": "Not_testing"
          }
        },
        {
          "lba_min": 0,
          "lba_max": 0,
          "status": {
            "value": 0,
            "string": "Not_testing"
          }
        },
        {
          "lba_min": 0,
          "lba_max": 0,
          "status": {
            "value": 0,
            "string": "Not_testing"
          }
        },
        {
          "lba_min": 0,
          "lba_max": 0,
          "status": {
            "value": 0,
            "string": "Not_testing"
          }
        },
        {
          "lba_min": 0,
          "lba_max": 0,
          "status": {
            "value": 0,
            "string": "Not_testing"
          }
        }
      ],
      "flags": {
        "value": 0,
        "remainder_scan_enabled": false
      },
      "power_up_scan_resume_minutes": 0
    }
  }
