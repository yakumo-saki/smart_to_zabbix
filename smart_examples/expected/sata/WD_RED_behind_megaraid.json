{
  "model": "WDC WD30EFRX-68EUZN0",
  "type": "sat+megaraid,0",
  "rotation_rate": 5400,
  "protocol": "ATA",
  "serial_number": "***************",
  "power_cycle": 750,
  "power_on_hours": 63823,
  "temperature": 39,
  "ssd.bytes_written": null,
  "ssd.lifespan": null,
  "smart_status_passed": 1
}
